README.md

```diff
@@ -50,6 +55,8 @@ of the files (`kernel.json`, `hass_pyscript_kernel.py`, `pyscript.conf`, and the
     - Since you've added a HASS access token to this file, you should make sure you are
       comfortable with file permissions - anyone who can read this file could use the
       access token to use the HASS UI without being an authenticated user.
+    - `hass_proxy` if you use SSH to access your HASS instance, you can open a SOCKS5 tunnel 
+      keep your Jupyter local. 
```

Comment:
This should explain what the setting actually is.  Also, is this missing "to" before "keep"?